cmd/api/src/auth/role_test.go

```diff
@@ -185,12 +185,12 @@ func testRoleAccess(t *testing.T, roleName string) {
 			}
 		}),
 
-		lab.TestCase(fmt.Sprintf("%s be able to access GraphDBWrite endpoints", testCondition(role, auth.Permissions().GraphDBWrite)), func(assert *require.Assertions, harness *lab.Harness) {
+		lab.TestCase(fmt.Sprintf("%s be able to access GraphDBIngest endpoints", testCondition(role, auth.Permissions().GraphDBIngest)), func(assert *require.Assertions, harness *lab.Harness) {
```

Comment:
As we've swapped this to GraphDBIngest, would you mind adding a GraphDBWrite test to this so we continue to have that coverate?

Reply:
Done. This was the trickiest part 😅